src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs

```diff
@@ -402,6 +403,16 @@ internal Metadata(MetadataExtractor extractor)
         /// <value>A string representing the information for 360 content, or null if the information does not exist.</value>
         public string Content360 { get; }
 
+        /// <summary>
+        /// Gets the information for stitched 360 content of the media.
```

Comment:
How does it work? A 360 video can be stitched or not, and this does not apply to any other media, right? Maybe a description like "Gets the information about 360 video content being stitched or not." or "Provides information whether a 360 video content is stitched."

Reply:
I'll change it as "Gets the information about 360 video content being stitched or not."